services/ozone/api.js

```diff
@@ -12,6 +12,11 @@ require('dd-trace') // Only works with commonjs
 
 // Tracer code above must come before anything else
 const path = require('path')
+const {
+  BunnyInvalidator,
+  CloudfrontInvalidator,
+  MultiImageInvalidator,
+} = require('@atproto/aws')
```

Comment:
We may need to adjust the service's package.json and dockerfile to include `@atproto/aws`.

Reply:
Easiest thing to check/catch this is to make a build and run it, even if it's unconfigured.

Reply:
yup yup good point 👍